client_firmware/client_firmware.ino

```diff
@@ -1,17 +1,18 @@
 #include <EEPROM.h>
 #include <EtherCard.h>
-#include <IPAddress.h>
 #include <avr/wdt.h>
 
 #define CSPIN 10
 // NOTE: All of these timers are in Milliseconds!
-// Ping our target every 2 seconds
-#define PINGER_INTERVAL 2000
+// Ping our target every 60 seconds
+#define PINGER_INTERVAL 60000
```

Comment:
I'm a little concerned here, on my branch I'm going to revert to 2s to support iphones (they wake up for ~10s every ~10m)

Did you notice it was draining batter or any bad effects?